### DISCUSSION TITLE
accepting multiple arguments in redux-toolkit in state

### DISCUSSION THREAD

**Discussion by ryzen322** [reduxjs/redux-toolkit#3011] [General]
here the createSlice 

export const todoSlice = createSlice({
  name: "todos",
  initialState: [],
  reducers: {
    addTodo: (state, action) => {
      // action.payload accept multiple arguments or object
      return [...state, action.payload];
    },
  },
});

export const { addTodo } = todoSlice.actions;

export default todoSlice.reducer;

// here 

  const [textName, setTextName] = useState("");
  const [text, setText] = useState("");

  const dispatch = useDispatch();

  const addTodoHandler = (e) => {
    e.preventDefault();

    
    dispatch(
      addTodo({
        id: nanoid(),
        name: textName,
        text: text,
      })
    );

    setTextName("");
    setText("");
  };

**Comment by phryneas** [comment]
If you have a question, please put in the effort to actually ask the question. That's not even a full sentence and nobody here can read your mind.

**Reply by ryzen322** [reply]
Noted mr @phryneas
